schema/body_schema.go

```diff
@@ -50,6 +51,24 @@ func NewBodySchema() *BodySchema {
 	}
 }
 
+func (as *BodySchema) SortAttributes() []string {
+	keys := make([]string, 0, len(as.Attributes))
+	for k := range as.Attributes {
+		keys = append(keys, k)
+	}
+	sort.Strings(keys)
+	return keys
+}
+
+func (as *BodySchema) SortBlocks() []string {
```

Comment:
```suggestion
func (as *BodySchema) BlockTypes() []string {
```
Similarly `SortedBlockTypes` would also be a reasonable name here, but we can also just imply that's sorted as most slices in Go should be anyway.